operator/main.go

```diff
@@ -369,7 +369,7 @@ func onOperatorStartLeading(ctx context.Context) {
 		// Start CEP watcher
 		operatorWatchers.CiliumEndpointsBatchInit(k8s.CiliumClient().CiliumV2(), cebController)
 		// Start the CEB controller, after current CEPs are synced locally in cache.
-		cebController.Run(operatorWatchers.CiliumEndpointStore, stopCh)
+		go cebController.Run(operatorWatchers.CiliumEndpointStore, stopCh)
```

Comment:
This looks like a bug fix, a separate commit might make more sense.

Reply:
Thats right, this is bug fix. Whole feature isn't in ready state, . Hopefully with this PR, CiliumEndpointBatch feature will be in working state. Going forward we can think about creating new PR for bug fixes.